### PR TITLE
fix(bridge): Token Vault sendEther messages with processing fees are impossible to send

### LINKED ISSUE #276: [bridge] bug - TokenVault unable to send message with processingFee > 0
in Bridge.sol when we check the .sendMessage call from TokenVault, we have this check:
```
        uint256 expectedAmount = message.depositValue +
            message.callValue +
            message.processingFee;
        require(expectedAmount == msg.value, "B:value");
```

however, when we call it from TokenVault.sendEther, we do this:
```
        message.gasLimit = gasLimit;
        message.processingFee = processingFee;
        message.depositValue = msg.value;
        message.refundAddress = refundAddress;
        message.memo = memo;

        // Ether are held by the Bridge on L1 and by the EtherVault on L2, not
        // the TokenVault
        bytes32 signal = IBridge(resolve("bridge")).sendMessage{
            value: msg.value
        }(message);

```

ie: it is imposible for msg.value to be equal to the message.deposit + call + processing amount, because depositValue on message is ALWAYS set to msg.value.

### PR DESCRIPTION
**Pull request by cyberhorsey**
Closes #276

### CODE REVIEW

File: packages/protocol/contracts/bridge/TokenVault.sol [hunk 1]
```diff
@@ -131,7 +131,7 @@ contract TokenVault is EssentialContract {
 
         message.gasLimit = gasLimit;
         message.processingFee = processingFee;
-        message.depositValue = msg.value;
+        message.depositValue = msg.value - processingFee;
```
[the remark this answers is not in the document]
Comment (dantaik) [reply packages/protocol/contracts/bridge/TokenVault.sol:134]: It means changing 
```
emit EtherSent(to, destChainId, msg.value, signal);
```
to
```
emit EtherSent(to, destChainId, msg.value - processingFee, signal);
```

**Review by dantaik — DISMISSED** [review]

File: packages/protocol/contracts/bridge/TokenVault.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (dantaik) [reply packages/protocol/contracts/bridge/TokenVault.sol:134]: Do we want to update the event as suggested above?

[the remark this answers is not in the document]
Comment (cyberhorsey) [reply packages/protocol/contracts/bridge/TokenVault.sol:134]: I'm not sure which is correct. Should the event show the amount the user will receive, or the total amount they sent?

[the remark this answers is not in the document]
Comment (dantaik) [reply packages/protocol/contracts/bridge/TokenVault.sol:134]: I think showing the amount to receive is better. The total amount of Ether used can also be retrieved from the same event.

[the remark this answers is not in the document]
Comment (dantaik) [reply packages/protocol/contracts/bridge/TokenVault.sol:134]: Let me change it for me then approve the PR.

**Review by dantaik — DISMISSED** [review]

File: packages/relayer/coverage.txt [hunk 2]
```diff
@@ -0,0 +1,151 @@
+mode: set
```
Comment (davidtaikocha) [inline packages/relayer/coverage.txt:1]: Need add this file to `.gitignore`? let me add it and then approve.

**Review by davidtaikocha — APPROVED** [review]

**Review by dantaik — APPROVED** [review]

**Pull request merged**